packages/components/steps/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import { CHANGE_EVENT } from '@element-plus/constants'
+import { buildProps, isNumber } from '@element-plus/utils'
+
+import type { ExtractPropTypes } from 'vue'
+
+export const stepsProps = buildProps({
+  space: {
+    type: [Number, String],
+    default: '',
+  },
+  active: {
+    type: Number,
+    default: 0,
+  },
+  direction: {
+    type: String,
+    default: 'horizontal',
+    validator: (val: string): boolean =>
+      ['horizontal', 'vertical'].includes(val),
+  },
+  alignCenter: {
+    type: Boolean,
+    default: false,
```

Comment:
We can remove `default: false` for `Boolean`

Simplified way:

```ts
{
  ...rest,
  alignCenter: Boolean,
}
```

Reply:
Thank you! your Opinion!